src/main/java/Duke.java

```diff
@@ -1,10 +1,37 @@
+import java.util.Objects;
+import java.util.Scanner;
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        Scanner reader = new Scanner(System.in);
+        Task[] items = new Task[100];
```

Comment:
Perhaps a constant could be introduced for the number 100 to avoid magic numbers?